src/test/java/bio/terra/janitor/integration/TrackResourceIntegrationTest.java

```diff
@@ -86,10 +94,14 @@ public void tearDownPubsub() throws Exception {
   @Test
   public void subscribeAndCleanupResource_googleBucket() throws Exception {
```

Comment:
per offline talk, let's have a test that checks that an already deleted bucket ends up in the DONE TrackedResourceState. Probably another PR.

Reply:
Added